src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -4,26 +4,38 @@
 
 public class FigureSupplier {
     private static final int LIMIT = 10;
+    private static final int CIRCLE_RADIUS = 10;
+    private static final int BOUND_FOR_SWITCH = 5;
 
     public Figure getRandomFigure() {
         ColorSupplier colorSupplier = new ColorSupplier();
         Random random = new Random();
-        Figure[] figures = new Figure[]
-                {new Circle(colorSupplier.getRandomColor(), random.nextInt(LIMIT)),
-                new RightTriangle(colorSupplier.getRandomColor(),
-                        random.nextInt(LIMIT), random.nextInt(LIMIT)),
-                new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
-                        random.nextInt(LIMIT), random.nextInt(LIMIT),random.nextInt(LIMIT)),
-                new Rectangle(colorSupplier.getRandomColor(),
-                        random.nextInt(LIMIT), random.nextInt(LIMIT)),
-                new Square(colorSupplier.getRandomColor(), random.nextInt(LIMIT))};
-
-        int index = new Random().nextInt(5);
-        return figures[index];
+        int index = new Random().nextInt(BOUND_FOR_SWITCH);
+        switch (index) {
+            case (0):
+                return new Circle(colorSupplier.getRandomColor(), random.nextInt(LIMIT));
+
+            case (1):
+                return new RightTriangle(colorSupplier.getRandomColor(),
+                        random.nextInt(LIMIT), random.nextInt(LIMIT));
+
+            case (2):
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        random.nextInt(LIMIT), random.nextInt(LIMIT),random.nextInt(LIMIT));
+
+            case (3):
+                return new Rectangle(colorSupplier.getRandomColor(),
+                        random.nextInt(LIMIT), random.nextInt(LIMIT));
+
+            case (4):
+                return new Square(colorSupplier.getRandomColor(), random.nextInt(LIMIT));
+
+            default:
+                return null;
+        }
```

Comment:
it's a bad practice to return null, let's use default case instead of case 4 to return one of the random figures. but don't forget to decrease BOUND_FOR_SWITCH by 1